### ISSUE TITLE
Element icons on details page

### ISSUE THREAD

**Issue by Zaczero** [openstreetmap-ng/openstreetmap-ng#58]
![image](https://github.com/Zaczero/openstreetmap-ng/assets/10835147/7a108bf3-03ac-4642-8414-0f66ec9d37e2)

![image](https://github.com/Zaczero/openstreetmap-ng/assets/10835147/8d010a45-52cd-4e29-bc62-d71478b9caa1)

The idea looks promising but the issue is that many icons are at very low resolutions, unsuitable for this use. Perhaps we could use some AI to enhance their quality?

**Comment by Aury88** [comment]
Hi, is it possible to use vector graphics instead of raster?If so I can try to help with this (I use Inkscape regularly).

**Comment by Zaczero** [comment]
Vector source is preferred (SVG). Unfortunately many of the current icons don't have one. Just make a pull request whenever you are ready 👍. Preferably we would want to start with upgrading the most popular icons first.

**Comment by Aury88** [comment]
I find only webp and png in app/static/img/*, are there other places where looking for?

**Comment by Zaczero** [comment]
One more folder: img/element

**Comment by Aury88** [comment]
ok, I was able to clone the repository, now I'm working on this. I will make a PR ASAP.

**Comment by Aury88** [comment]
ok,I  starded doing the first icons:
accommodation_youth_hostel.svg

-  aeroway_aerodrome.2.svg
-  aeroway_aerodrome.svg
-  aeroway_airport.2.svg
-  aeroway_airport.svg
-  aeroway_gate.svg
-  aeroway_helipad.svg
-  aeroway_runway.20.svg
-  aeroway_taxiway.20.svg
-  aeroway_terminal.svg
-  amenity_atm.16.svg
-  amenity_atm.svg
-  amenity_bank.16.svg
-  amenity_bank.svg
-  amenity_bar.16.2.svg
-  amenity_bar.16.svg
-  amenity_bar.2.svg
-  amenity_bar.svg
-  amenity_bench.16.svg
-  amenity_bench.svg
-  amenity_bicycle_parking.svg
-  amenity_bicycle_rental.16.svg
-  amenity_bicycle_rental.svg
-  amenity_biergarten.16.svg
-  amenity_bureau_de_change.svg
-  amenity_bus_station.16.svg
-  amenity_bus_station.svg
-  amenity_cafe.16.svg
-  amenity_cafe.2.svg
-  amenity_cafe.svg
-  amenity_car_rental.svg
-  amenity_car_sharing.2.svg
-  amenity_car_sharing.svg
-  amenity_casino.svg
-  amenity_cinema.16.svg
-  amenity_cinema.svg
-  amenity_clock.svg
-  amenity_college.svg
-  amenity_courthouse.16.svg
-  amenity_courthouse.svg
-  amenity_dentist.16.2.svg
-  amenity_dentist.16.svg
-  amenity_dentist.svg
-  amenity_doctors.16.2.svg
-  amenity_doctors.16.svg
-  amenity_doctors.svg
-  amenity_drinking_water.16.svg
-  amenity_drinking_water.svg
-  amenity_fast_food.16.2.svg
-  amenity_fast_food.16.svg
-  amenity_fast_food.svg
-  amenity_fire_station.16.svg
-  amenity_fire_station.svg
-  amenity_fountain.svg
-  amenity_fuel.16.svg
-  amenity_fuel.svg
-  amenity_hospital.16.svg
-  amenity_hospital.svg
-  amenity_hospital_emergency.svg
-  amenity_ice_cream.2.svg
-  amenity_ice_cream.svg
-  amenity_library.16.svg
-  amenity_library.svg
-  amenity_marketplace.svg
-  amenity_nightclub.16.svg
-  amenity_nursery.svg
-  amenity_parking.svg
-  amenity_parking_car.svg
-  amenity_parking_disabled.svg
-  amenity_parking_fee.svg
-  amenity_parking_private.svg

now, having had problem in the past I tried a first PR before going on but the comand

 `git request-pull https://github.com/openstreetmap-ng/openstreetmap-ng.git`

doesn't work. ...probably I done  a so noob error (this is my first PR, always had to push in the past)  that I  was not able to find the solution  on google.
can someone give me instruction? thank you

**Comment by KonradowyPL** [comment]
You have to fork repository first. Then set origin url via `git remote set-url https://github.com/something` then push the changes and finally make the PR.

**Comment by Zaczero** [comment]
Since we have started working on this issue, I have assigned a bounty to it.

**Comment by Zaczero** [comment]
> is better one final huge PR with all commits

Since we are not in a hurry, we can wait for the changes to accumulate and do just 1 PR.

I can see that your fork is 3 commits ahead currently: https://github.com/Aury88/openstreetmap-ng/commits/main/

The last things seems like a bug.
When I click on the "0 commits" I can see the commit you have made.
Maybe it will fix itself after some time.

**Comment by Aury88** [comment]
Small update:  there are 89 icons left to remake (done 327). from tomorrow my holidays ends so I can work on this from now on only during the weekends. I hope to finish by September.
bye

**Comment by Aury88** [comment]
Hi all.
On this issue I'm having some doubts. There are icons that I think can be replaced with much clearer and prettier ones. The doubt is this: have these icons  to be the same as the symbols shown on the map or can they be different?
if they can be different I would like to try  to change some of them with something different.

**Comment by Zaczero** [comment]
@Aury88 Hey sorry for the delayed response - I am more busy than usual this week. I think we don't mind changing the icons as long as they are for the better.

**Comment by Aury88** [comment]
don't worry.  for me this was a busy too.
 I've done a PR but failed two tests...I don't know why...they probably are too huge.
tell me if I can/need to do something about that.

**Comment by Aury88** [comment]
so is there any news about this issue and the last PR failed Check?  Can I do something?

**Comment by Zaczero** [comment]
> so is there any news about this issue

I have been sick for the past few days and just now I'm getting back to programming and stuff.

> and the last PR failed Check

It is unrelated to your work, but it's still weird that it's started to fail. I'll fix it.

**Comment by Zaczero** [comment]
@Aury88 Since this is your first bounty, you can keep collecting the funds and let me know whenever you would like to withdraw. It's tracked [here](https://github.com/Zaczero/openstreetmap-ng/wiki/Ledger). I have decided to assign you 30$ instead of 10$ for your half of the work, since I believe this was a tedious process and you have shown some real dedication. This work will have great positive impact on the project!

**Comment by Aury88** [comment]
hi,
I'm sorry to hear you've been sick. I hope everything is okay now. I wrote only because having only weekends available (and not entirely) the risk was that if there was something to fix I wouldn't have been able to do it for at least a week.
yes I didn't expect so many icons when I volunteered and it was sometime stressful considering I had set myself to finish everything in a couple of weeks, but overall I had fun. About the bounty my intention is to give back the money  as a donation to the OSMng project. can we do this directly so to avoid transaction fees? Bye
